dev-client/src/translations/en.json

```diff
@@ -76,13 +76,40 @@
         "action_name": "Delete"
       },
       "location_accuracy": "Location Accuracy",
-      "default_title": "Location Information",
+      "default_title": "Temporary Pin",
```

Comment:
issue: this sounds like an internal name, not something user-visible.

Reply:
(comment: i continue to feel that the right place to give feedback on copy is in the Figma, not in PRs, since it's unrelated to implementation)

Reply:
I left a comment in the Figma to this effect, but will move forward with this PR unless I hear otherwise